python/GangaLHCb/Lib/Splitters/GaussSplitter.py

```diff
@@ -39,39 +42,59 @@ def _create_subjob(self, job, inputdata):
         return j
 
     def split(self, job):
+        """
+            Method to do the splitting work
+            Args:
+                job (Job): master job to be used as a template to split subjobs
+        """
+
+        # NB Not sure if we can ever _not_ import the Gauss app from the GPI
+        from Ganga.GPI import Gauss
```

Comment:
You should be able to do:
```python
from GangaLHCb.Lib.Applications import Gauss
```
Though looking at the code that generates that name (`GangaLHCb/Lib/Application/__init__.py`) is not recommended :unamused: